qa/rpc-tests/wallet_shieldingcoinbase.py

```diff
@@ -107,7 +107,7 @@ def run_test (self):
         assert_equal(error_result["method"], "z_sendmany")
         params = error_result["params"]
         assert_equal(params["fee"], DEFAULT_FEE) # default
-        assert_equal(params["minconf"], Decimal('1')) # default
+        assert_equal(params["minconf"], Decimal('10')) # default
```

Comment:
```suggestion
        assert_equal(params["minconf"], Decimal('3')) # default
```

Reply:
The default minconf is staying as 10; only the default anchor confirmations will be set to 3.